engines/dfc_provider/lib/data_food_consortium/connector/importer.rb

```diff
@@ -127,11 +127,14 @@ def skos_concept(object)
       end
 
       def guess_setter_name(predicate)
-        fragment = predicate.fragment
+        name =
 
```

Comment:
Breaking a command over empty lines feels wrong to me, so what you think about removing the empty lines?

```suggestion
```

Reply:
Okay, done. I was wondering that, too. There's another style rule to have blank lines above comments but rubocop doesn't complain here. So all good.